andela/team-odd-bn-frontend
# #169258576 User should be able to reset password

#### What does this PR do?
- This PR adds password reset functionalities for the user
#### Description of Task to be completed?
A user will be presented with a button and or link to reset their passwords on the SignUp/Register component:
- A page for a user to provide his/her email
- A page to insert new password
#### How should this be manually tested?

1. Run `git clone https://github.com/andela/team-odd-bn-frontend.git`
2. Run `cd team-odd-bn-frontend`
3. Run `git checkout ft-reset-password-link-169258576`
4. Run `npm install` to install all dependencies
5. Run `npm run dev` to start development server
6. A page should open automatically from your default browser
7. Enter email address 
8. Click send reset request button
9. Check your email to open new password setting page

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169258576](https://www.pivotaltracker.com/story/show/169258576)
#### Screenshots (if appropriate)
###### Actual  forgot password page
<img width="1440" alt="Screen Shot 2020-01-05 at 13 17 48" src="https://user-images.githubusercontent.com/35301129/71779235-0442cd80-2fbe-11ea-919a-e66c409858f6.png">

###### Responsive forgot password page
<img width="141" alt="Screen Shot 2020-01-05 at 13 17 34" src="https://user-images.githubusercontent.com/35301129/71779234-03aa3700-2fbe-11ea-8bf3-28bf19c19eb0.png">

###### Actual  reset password page
<img width="1440" alt="Screen Shot 2020-01-05 at 13 18 12" src="https://user-images.githubusercontent.com/35301129/71779236-04db6400-2fbe-11ea-93d6-c863ea296436.png">

###### Responsive reset password page
<img width="144" alt="Screen Shot 2020-01-05 at 13 18 53" src="https://user-images.githubusercontent.com/35301129/71779237-0573fa80-2fbe-11ea-81a6-8beea485e85d.png">

#### Questions:
N/A
